packages/gestalt/src/ComboBox.js

```diff
@@ -337,6 +337,7 @@ const ComboBoxWithForwardRef: React$AbstractComponent<Props, HTMLInputElement> =
                     onSelect={({ event, item }) => handleSelectOptionItem({ event, item })}
                     selected={selectedOption ?? selectedItem}
                     setHoveredItemIndex={setHoveredItemIndex}
+                    truncate={2}
```

Comment:
I think I'm missing something, but why 2 here and 1 elsewhere?

Reply:
@ayeshakmaz So, I could do 1... The issue here is that they want this combobox in very narrow width field... so one line would show next to no text.  Adding 2 line clamp could give a little bit more context for each line. But I can keep 1 for consistency

![image](https://user-images.githubusercontent.com/10593890/140386278-ec879431-69fe-4f41-9f43-968e9c9f08db.png)